source/Esf.Website/App/common/models/esfStateViewModel.ts

```diff
@@ -0,0 +1,29 @@
+﻿import { EsfStateDto } from './esfStateDto';
```

Comment:
We should move away things from the common folder and group by them feature. Most of the things here are either for query running or for esf State